packages/nestjs-logging-interceptor/README.md

```diff
@@ -0,0 +1,126 @@
+<p align="center">
+  <a href="http://nestjs.com"><img src="https://nestjs.com/img/logo_text.svg" alt="Nest Logo" width="320" /></a>
+</p>
+
+<p align="center">
+  A <a href="https://github.com/nestjs/nest">Nest</a> interceptor to log the incoming/outgoing requests.
+</p>
+
+# NestJS Logging interceptor
+
+A simple NestJS interceptor catching request details and loggin it using the built-in [Logger](https://docs.nestjs.com/techniques/logger#logger) class. It will use the default Logger implementation unless you pass your own to your Nest application.
+
+## Installation
+
+```bash
+npm install --save @algoan/nestjs-logging-interceptor
+```
+
+## Usage
+Use the interceptor as a global interceptor (cf. refer to the [last paragraph](https://docs.nestjs.com/interceptors#binding-interceptors) of this section for more details).
```

Comment:
Could you give an example with another logger? Like [nestjs-pino](https://github.com/iamolegga/nestjs-pino) for instance